.vale.ini

```diff
@@ -1,7 +1,4 @@
 StylesPath = "styles/camunda"
 
 [{docs,versioned_docs}/**/*.{md,mdx}]
-BasedOnStyles = all, docsInstance
```

Comment:
We had a set of rules for only the main instance, and a set for only the optimize instance. I moved all valid rules back into the `all` folder, since all rules now apply to all docs.